### ISSUE TITLE
Lock: couldn't read LoopLock response: unexpected EOF

### ISSUE THREAD

**Issue by jkawamoto** [lukechampine/us#162]
Even if we choose a longer timeout limit, many hosts return this error.
```
3e37b161: Lock: couldn't read LoopLock response: unexpected EOF 2m55.025174454s
eedb600b: Lock: couldn't read LoopLock response: unexpected EOF 3m10.169237825s
ca57de76: Lock: couldn't read LoopLock response: unexpected EOF 3m10.28172274s
5b18d71d: Lock: couldn't read LoopLock response: unexpected EOF 3m14.118610765s
e9069491: Lock: couldn't read LoopLock response: unexpected EOF 3m14.119345141s
d5bf5774: Lock: couldn't read LoopLock response: unexpected EOF 5m0.933609978s
fdaa28bc: Lock: couldn't read LoopLock response: unexpected EOF 5m0.947730759s
1c96a10c: Lock: couldn't read LoopLock response: unexpected EOF 5m1.045985658s
0a0c32a6: Lock: couldn't read LoopLock response: unexpected EOF 5m1.21115166s
c32db729: Lock: couldn't read LoopLock response: unexpected EOF 5m1.328821334s
7a088582: Lock: couldn't read LoopLock response: unexpected EOF 5m1.494179386s
d304d705: Lock: couldn't read LoopLock response: unexpected EOF 5m1.503361754s
a9f3209e: Lock: couldn't read LoopLock response: unexpected EOF 5m2.066356114s
51032222: Lock: couldn't read LoopLock response: unexpected EOF 5m2.159676065s
```

I remember hosts have a shorter timeout limit (around 2min?). Can we do anything to avoid this error? We don't mind locking slower hosts because overdrive ends up choosing faster hosts.

**Comment by lukechampine** [comment]
Looks like the host always sets a deadline of 2 mins, even if the lock timeout is longer. I'm not 100% sure this is the cause of the bug, but fixing it wouldn't hurt. I'll submit a patch.

**Comment by lukechampine** [comment]
This is being released in `siad` v1.5.6
